setuptools/unicode_utils.py

```diff
@@ -65,10 +65,10 @@ def _read_utf8_with_fallback(file: str, fallback_encoding=py39.LOCALE_ENCODING)
 
 
 def _cfg_read_utf8_with_fallback(
-    cfg: ConfigParser, file: str, fallback_encoding=py39.LOCALE_ENCODING
+    cfg: RawConfigParser, file: str, fallback_encoding=py39.LOCALE_ENCODING
 ) -> None:
     """Same idea as :func:`_read_utf8_with_fallback`, but for the
-    :meth:`ConfigParser.read` method.
+    :meth:`RawConfigParser.read` method.
```

Comment:
This also allows any subclass, right?

Reply:
Right. Setuptools passes a `RawConfigParser`, but `ConfigParser`, which is a subclass, will be statically allowed.